mne/fixes.py

```diff
@@ -475,6 +477,438 @@ def _get_sph_harm():
 
 
 ###############################################################################
+# Scipy spectrogram (for mne.time_frequency.psd_welch) needed for scipy < 0.16
+
+
+
+def _spectrogram(x, fs=1.0, window=('tukey',.25), nperseg=256, noverlap=None,
+                nfft=None, detrend='constant', return_onesided=True,
+                scaling='density', axis=-1, mode='psd'):
```

Comment:
yes - these defaults are unchanged, but are called with different window from `_pwelch` so that the options are the same as before.